src/EventHandlers/onmemberjoin.py

```diff
@@ -0,0 +1,12 @@
+import discord 
+notification_channel_name = 'botplayground'
+member_notification_number = 100
+
+
+async def on_member_join(bot,member):
```

Comment:
I would check the docs to see whether or not the memberjoin event provides extra data that we can use to determine the guild the new member joined.

now you check this for every guild the bot is in.
This means that when we get 2000 members we get a notification
But if a new member joins another guild, this code will run again, see that we have 2000 members and show the message again :)

Reply:
Changed to `guild = member.guild` to fetch the guild where the new member has joined. That should solve the issue.